src/components/QRCodeModal/index.tsx

```diff
@@ -0,0 +1,84 @@
+import QRCode from 'qrcode.react';
+import React, { useEffect, useRef } from 'react';
+import { FcDownload } from 'react-icons/fc';
+import { IoCloseSharp } from 'react-icons/io5';
+
+import Button from '@/components/Button';
+const RDSIcon = '_next/image?url=%2Frds.png&w=64&q=75';
+
+interface QRCodeModalProps {
+    value: string;
```

Comment:
value can be renamed to be more specific, ex. `shortUrl`

Reply:
Ok